spec/features/user/profile_spec.rb

```diff
@@ -0,0 +1,110 @@
+require 'rails_helper'
+
+RSpec.describe "User profile page" do
+  before(:each) do
+    @regular_user = User.create(name: "Willy Wonka", address: "123 St", city: "Denver", state: "CO", zip: "12345", email: "chocolateguy1@gmail.com", password: "loco123", role: 0)
```

Comment:
Shout out to Danny for this seed.

Reply:
squaaaaad

Reply:
![](https://media1.giphy.com/media/1132uJKzZQc4ow/giphy.gif?cid=5a38a5a2f9f73c95941f1306a753749d38fbd6b439d9ff34&rid=giphy.gif)